src/components/Layout.jsx

```diff
@@ -70,4 +70,10 @@ class TemplateWrapper extends Component {
   }
 }
 
-export default TemplateWrapper;
+export const Head = () => (
+  <>
+    <link href='https://fonts.gstatic.com' rel='preconnect' crossorigin/>
```

Comment:
it seems that this has no effect. Reading this article https://www.igvita.com/2015/08/17/eliminating-roundtrips-with-preconnect/

Reply:
From Lighthouse test: `Warnings: A `<link rel=preconnect>` was found for "http://cdnjs.cloudflare.com" but was not used by the browser. Only use `preconnect` for important origins that the page will certainly request.`
So maybe better to remove this?